circuit_maintenance_parser/parser.py

```diff
@@ -408,6 +409,17 @@ def _get_circuit_ids(self, generated_json: dict, impact: Impact):
 
         return circuits
 
+    def _convert_str_datetime_to_epoch(self, time_str):
+        try:
+            # Using isoparse for flexible ISO8601 parsing
+            time_dt = isoparse(time_str).astimezone(datetime.timezone.utc)
+            epoch_time = int(time_dt.timestamp())
+            return epoch_time
+
+        except Exception as error_received:
+            logger.debug("Error parsing dates: %s", error_received)
```

Comment:
I think it deserves an error, no?
```suggestion
            logger.error("Error parsing dates: %s", error_received)
```